po-admin/index.php

```diff
@@ -34,50 +34,57 @@
+    <script type="text/javascript">
+        var WEB_URL = "<?=WEB_URL;?>";
+        var DIR_INC = "<?=DIR_INC;?>";
+        var DIR_CON = "<?=DIR_CON;?>";
+        var DIR_ADM = "<?=DIR_ADM;?>";
+    </script>
 
-	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/jquery/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/bootstrap/bootstrap.min.js"></script>
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
 </head>
 <body>
-	<section id="login">
-		<div class="container">
-			<div class="row">
-				<div class="col-md-12">
-					<div class="form-wrap">
-						<div class="col-md-12 text-center">
-							<img src="../<?=DIR_INC;?>/images/logo.png" class="logo" width="100" />
-						</div>
-						<?php
-							if (VQMOD == TRUE) {
-								include_once VQMod::modCheck("route.php");
-							} else {
-								include_once "route.php";
-							}
-						?>
-						<div class="col-md-12 text-center" id="footer">
-							<p><?=CONF_STRUCTURE;?> &copy; 2013-2016. MIT License</p>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-	</section>
-	<script type="text/javascript" src="../<?=DIR_INC;?>/js/patternlock/patternLock.js"></script>
-	<script type="text/javascript" src="../<?=DIR_INC;?>/js/login-core.js"></script>
+    <section id="login">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="form-wrap">
+                        <div class="col-md-12 text-center">
+                            <img src="../<?=DIR_INC;?>/images/logo.png" class="logo" width="100" />
+                        </div>
+                        <?php
+                            if (VQMOD == TRUE) {
+                                include_once VQMod::modCheck("route.php");
+                            } else {
+                                include_once "route.php";
+                            }
+                        ?>
+                        <div class="col-md-12 text-center" id="footer">
+                            <p><?=CONF_STRUCTURE;?> &copy; 2013-2016. MIT License</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/patternlock/patternLock.js"></script>
+    <script type="text/javascript" src="../<?=DIR_INC;?>/js/login-core.js"></script>
 </body>
 </html>
 <?php
 } else {
-	header('location:admin.php?mod=home');
+    header('location:admin.php?mod=home');
 }
 ?>
```

Comment:
> Ups...
> sepertinya saya lupa mematikan auto converter tab ke spasi...

I put standarized php coding style (PSR-4) configuration in my sublimetext. And I forgot to turn it all off. Maintainer seems like doesn't follow it at all.
I will be more carefull next time.